crates/pathfinder/src/state/sync/l2.rs

```diff
@@ -140,27 +137,9 @@ where
             {
                 DownloadBlock::Block(block, commitments) => break (block, commitments),
                 DownloadBlock::AtHead => {
-                    // Poll pending if it is enabled, otherwise just wait to poll head again.
-                    match pending_poll_interval {
-                        Some(interval) => {
-                            tracing::trace!("Entering pending mode");
-                            let head = head_meta
-                                .expect("Head hash should exist when entering pending mode");
-                            (next_block, next_state_update) = pending::poll_pending(
-                                tx_event.clone(),
-                                &sequencer,
-                                (head.1, head.2),
-                                interval,
-                                storage.clone(),
-                            )
-                            .await
-                            .context("Polling pending block")?;
-                        }
-                        None => {
-                            tracing::info!(poll_interval=?head_poll_interval, "At head of chain");
-                            tokio::time::sleep(head_poll_interval).await;
-                        }
-                    }
```

Comment:
Ah I should have been more specific. The pending polling itself should remain - it just shouldn't be configurable from outside. Should just always be 2s set from main.